File: test_emailjs.js
const data = {
  service_id: 'service_25libn5',
  template_id: 'template_tu878js',
  user_id: 'o3tsRNH8slV7K4jTP',
  template_params: {
    name: 'Test User',
    title: 'Test Radicado RAD-12345',
    email: 'test@example.com'
  }
};

fetch('https://api.emailjs.com/api/v1.0/email/send', {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36'
  },
  body: JSON.stringify(data)
})
.then(async res => {
  const text = await res.text();
  console.log('Status:', res.status);
  console.log('Response:', text);
  if (res.status === 200) {
    console.log('EmailJS is working correctly!');
  } else {
    console.log('EmailJS is NOT working correctly.');
  }
})
.catch(err => {
  console.error('Error:', err);
});

File: test_emailjs.py
import urllib.request
import json
import sys

def test_emailjs():
    url = 'https://api.emailjs.com/api/v1.0/email/send'
    payload = {
        'service_id': 'service_25libn5',
        'template_id': 'template_tu878js',
        'user_id': 'o3tsRNH8slV7K4jTP',
        'template_params': {
            'name': 'Test User',
            'title': 'Test Radicado RAD-12345',
            'email': 'test@example.com'
        }
    }
    
    data = json.dumps(payload).encode('utf-8')
    req = urllib.request.Request(url, data=data, headers={'Content-Type': 'application/json'}, method='POST')
    
    try:
        with urllib.request.urlopen(req) as response:
            body = response.read().decode('utf-8')
            print(f"Status: {response.status}")
            print(f"Response: {body}")
            if response.status == 200:
                print("EmailJS is working correctly!")
            else:
                print("EmailJS is NOT working correctly. Returned non-200 status.")
    except urllib.error.HTTPError as e:
        print(f"HTTP Error: {e.code}")
        print(f"Response: {e.read().decode('utf-8')}")
        print("EmailJS is NOT working correctly.")
    except Exception as e:
        print(f"Error: {e}")
        print("EmailJS is NOT working correctly.")

if __name__ == "__main__":
    test_emailjs()
